cmd/init_project.go

```diff
@@ -68,21 +68,23 @@ kubebuilder init --domain example.org --license apache2 --owner "The Kubernetes
 }
 
 type projectOptions struct {
-
-	// flags
-	fetchDeps          bool
-	skipGoVersionCheck bool
-
 	boilerplate project.Boilerplate
 	project     project.Project
 
-	// deprecated flags
-	dep     bool
+	// final result
+	scaffolder scaffold.ProjectScaffolder
+
+	// @Deprecated
 	depFlag *flag.Flag
+	// @Deprecated
 	depArgs []string
 
-	// final result
-	scaffolder scaffold.ProjectScaffolder
+	// flags
+	fetchDeps          bool
+	skipGoVersionCheck bool
+
+	// @Deprecated
+	dep bool
```

Comment:
Why split this from the other deprecated flags?

Reply:
In order to solve the `maligned` issue check, I need to to add the bools at the end. 
This is about memory space usage.